File: UNIDAD 1/Numero_par.py
#Desarrolle un programa que permita encontrar si un numero ingresado es par o impar.
#Variables
num = 0
num = int(input("Ingrese un nuermo entero: "))

if num < 0:
    print("El numero ingresado debe ser positivo")
else:
    if num/2*2 ==num:
        #num % 2 == 0:
        print("El numero leido es par")
    else:
      print("El numero es Impar")

File: numero_par.cpp
#include <iostream.h>
void main()
	int num;
	cin >> num;
	if (num < 0)
		cout<<"El numero debe ser positivo";
	else
		if(num/2*2==num)
			cout<< "El numero es par ";
		else
			cout<<"El numero es Impar";
